bpf/lib/trace.h

```diff
@@ -141,6 +141,12 @@ static __always_inline bool emit_trace_notify(__u8 obs_point, __u32 monitor)
 		}
 	}
 
+	/*
+	 * Ignore sample when aggregation is enabled and monitor is 0.
+	 * Trace points that exist outside of CT context cannot do rate limiting,
+	 * so they frequently set monitor to 0 to be ignored when aggregation is
+	 * enabled (the default).
```

Comment:
nit, I suspect that when you say "trace points" here you are referring to the BPF code in the tree which calls `send_trace_notify()`. However, "trace points" in BPF context has a complete other meaning relating to where in the kernel that the BPF program is attached, which makes the comment ambiguous.

Also, we often use the word "context" in BPF to refer to the parameters passed into a BPF hook, so reusing that word feels a bit imprecise to me.

Here's a couple of alternate suggestions that I think would still be correct and may provide the additional context that will help future readers:

```suggestion
	 * Hook points that do not perform connection tracking cannot do rate limiting,
	 * so they frequently set monitor to 0 to be ignored when aggregation is
	 * enabled (the default).
```

Or with fewer negative statements,

```suggestion
	 * Rate limiting relies on connection tracking, so if the BPF program does
	 * not perform CT then monitor will be set to 0 to avoid emitting trace
	 * notifications when aggregation is enabled (the default).
```

Reply:
> "trace points" in BPF context

> the word "context" in BPF

Absolutely, that would indeed be confusing.. In lieu of what I mean here by 'trace points', we generally use 'observation points' throughout the codebase, I would consider that to be the most accurate description.

Reply:
I've made the following change in b189308:

```
+     * Ignore sample when aggregation is enabled and 'monitor' is set to 0.
+     * Rate limiting (trace message aggregation) relies on connection tracking,
+     * so if there is no CT information available at the observation point,
+     * then 'monitor' will be set to 0 to avoid emitting trace notifications
+     * when aggregation is enabled (the default).
```

I've also clarified what is meant by 'rate limiting'. WDYT? @joestringer

Reply:
:ok_hand: perfect.